index.html

```diff
@@ -35,8 +35,8 @@ <h3>
                     </a>
                 </li>
                 <li>
-                    <a href="#footer-container-id" class="link-contact">
-                        Contact
+                    <a href="#footer-container-id" class="link-contact"><p>
+                        Contact <p/>
                     </a>
```

Comment:
[OPTIONAL] 
- [ ] Kindly close your paragraph tag correctly.
```suggestion
                    <a href="#footer-container-id" class="link-contact"><p>
                        Contact </p>
                    </a>
```